dawn/src/dawn/Unittest/CMakeLists.txt

```diff
@@ -19,4 +19,5 @@ yoda_add_library(
           IIRBuilder.cpp
           IIRBuilder.h
   ARCHIVE
+  DEPENDS DawnIIRStatic DawnOptimizerStatic
```

Comment:
I am not sure this is the correct way to define the dependencies in the yoda framework, but Unittest needs to resolve symbols from DawnIIR and DawnOptimizer.

Reply:
@cosunae